examples/with_service_discovery/test.tfvars

```diff
@@ -1,6 +1,7 @@
 logical_product_service = "dso102"
 # Ensure you have a profile by this name in your ~/.aws/config file
 aws_profile = "launch-sandbox-admin"
+app_image   = "<replace>"
```

Comment:
I think we need a real app image here, or terratest will fail when it applies this example.

Reply:
@bryce-lynn-nttd , you should be aware that there are other placeholder values within angle brackets in other module repos. 

@aaron-christian-nttd , you might be able to use "public.ecr.aws/nginx/nginx:latest" as a placeholder here